supabase/migrations/20230606180645_add_bookmarked_projects.sql

```diff
@@ -0,0 +1,24 @@
+create table public.user_bookmarks (
+    "project" text not null,
+    "user_id" uuid not null,
+    "wallet" char(42) not null,
```

Comment:
could we derive wallet from the users table instead? in other words, do we need this column?